helix-view/src/editor.rs

```diff
@@ -313,6 +313,9 @@ pub enum StatusLineElement {
     /// The file nane/path, including a dirty flag if it's unsaved
     FileName,
 
+    /// The file absolute path, including a dirty flag if it's unsaved
+    FileAbsPath,
```

Comment:
This is quite a small nitpick but could we call this `FileAbsolutePath` (and `file_absolute_path` for the function)? With the other elements we haven't done any abbreviation yet so I think we should keep it consistent

Reply:
Makes sense to be consistent.  Updated.